src/wwwroot/services/templates.js

```diff
@@ -19,7 +19,8 @@
       getAllData: getAllData,
       getTemplate: getTemplate,
```

Comment:
Maybe this method could be removed since it should not be being used anymore.

Reply:
Yes, I forget it. thx